shared/src/test/diff/qq/Nested.mls

```diff
@@ -0,0 +1,170 @@
+:NewDefs
+:NoJS
+
+
+:e
+let t = code"id(code"123")"
+//│ ╔══[ERROR] Nested quotation is not allowed.
+//│ ║  l.6: 	let t = code"id(code"123")"
+//│ ╙──     	                ^^^^^^^^^
+//│ let t: Code[error, nothing]
+
+let inner0 = code"123"
+let t = code"id(inner0)"
+//│ let inner0: Code[123, nothing]
+//│ let t: Code[Code[123, nothing], nothing]
```

Comment:
BTW I just found that we can still have nested `Code` types via top-level definitions. Should we have a check for this situation?

Reply:
I added this test and I don't know what you think is wrong with it. What is your suggestion meant to accomplish?

Reply:
I thought we did not support nested codes. But this case would not lead to level violation so I guess it is fine.

Reply:
We don't support nested quasiquotes.

There is nothing special about nested code types.